components/brave_new_tab_ui/components/default/page/index.tsx

```diff
@@ -315,12 +315,14 @@ function getBackground(p: HasImageProps) {
   }
 
   if (p.hasImage && p.imageSrc) {
+    // Note: We force percent encoding for ( and ) because Chromium seems to be
+    // ignoring the fact that the URL is quoted for these.
     return `linear-gradient(
       rgba(0, 0, 0, 0.8),
       rgba(0, 0, 0, 0) 35%,
       rgba(0, 0, 0, 0) 80%,
       rgba(0, 0, 0, 0.6) 100%
-    ), url("${p.imageSrc}")`
+    ), url("${p.imageSrc.replaceAll('(', '%28').replaceAll(')', '%29')}")`
```

Comment:
Wow, didn't know that `encodeURIComponent` doesn't convert parenthesis. I'd checked space but missed this one.

Reply:
I know right, it's random as! Additionally, I would've thought having the url in quotes would be enough to no break the parsing:

`background-image: url("https://foo.com/(haha)")` breaks though! Interestingly, it sets the background properly, but then updates the url to be something nonsensical :laughing: